pkg/controller/configmap/config.go

```diff
@@ -158,6 +168,16 @@ func (adaptor *ConfigMapOc) validateConfigMap(configmap *corev1.ConfigMap) []err
 	if err != nil {
 		appendErrorIfNotNil(&errs, errors.Wrapf(err, "mtu is invalid"))
 	}
+	// Check DEFAULT section log_file option
```

Comment:
this part is the same as 132-141, maybe add a method instead of same code block.

Reply:
We trying to merge the two func (adaptor *ConfigMapOc) validateConfigMap(configmap *corev1.ConfigMap) and func (adaptor *ConfigMapK8s) validateConfigMap(configmap *corev1.ConfigMap) since the logic of these two functions are basically the same. but it's a little bit rush to do it in the operator 4.1.0. We can consider this in the next release.